manager/controllers/app/config/evaluator_interface.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import corev1 "k8s.io/api/core/v1"
+
+// EvaluatorInterface is an interface for config policies' evaluator
+type EvaluatorInterface interface {
+	Evaluate(in *EvaluatorInput) (EvaluatorOutput, error)
```

Comment:
Should there also be an interface for loading the relevant config policies?  For now its implementation would do nothing.

Reply:
It should, I thought to add when it's ready and replaces the default implementation in manager.